hedera-node/hedera-mono-service/src/main/java/com/hedera/services/txns/crypto/CryptoCreateTransitionLogic.java

```diff
@@ -119,6 +132,26 @@ public void doStateTransition() {
 
             txnCtx.setCreated(created);
             txnCtx.setStatus(SUCCESS);
+            if (dynamicProperties.isCryptoCreateWithAliasEnabled()
+                    && dynamicProperties.isLazyCreationEnabled()) {
```

Comment:
Won't we miss linking the alias with the new account in the case where we have `cryptoCreateWithAlias` true, `lazy create` = false, and we submit `CryptoCreate` with public key alias?